packages/plugin-lighthouse/src/lib/lighthouse-plugin.ts

```diff
@@ -1,25 +1,31 @@
-import { AuditOutputs, PluginConfig } from '@code-pushup/models';
+import { Audit, AuditOutputs, PluginConfig } from '@code-pushup/models';
+import { filterByAuditSlug, filterBySlug } from '@code-pushup/utils';
 import { AUDITS, GROUPS, LIGHTHOUSE_PLUGIN_SLUG } from './constants';
 
+type AuditSlug = (typeof AUDITS)[number]['slug'];
```

Comment:
If it is necessary to check that the value is a slug, it would imo be better to use a slugSchema. Otherwise, a `string` should suffice or what am I missing?

Reply:
Failed attempt at inferring a union, I believe: https://github.com/code-pushup/cli/pull/472#discussion_r1480107285